tests/test_cipher.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# # -*- coding: utf-8 -*-
+
+"""
+Unit tests for cipher.
+________________________________________________________________________________
+
+Created by brightSPARK Labs
+www.brightsparklabs.com
+"""
+
+# vendor libraries
+import pytest
+from pathlib import Path
+
+# local libraries
+from appcli.crypto import crypto
+from appcli.crypto.cipher import Cipher
+
+
+# ------------------------------------------------------------------------------
+# TESTS
+# ------------------------------------------------------------------------------
+
+
+def test_aes_gcm(tmpdir):
+    cipher = create_cipher(tmpdir)
+
+    plaintext = "a message to encrypt"
+    ciphertext = cipher.encrypt(plaintext)
+    assert ciphertext != plaintext
+    result = cipher.decrypt(ciphertext)
+    assert result == plaintext
+
+
+def test_missing_prefix(tmpdir):
+    cipher = create_cipher(tmpdir)
+    with pytest.raises(Exception) as ex:
+        cipher.decrypt(":id=aes-gcm:data:end")
+    assert "Encrypted data must have format [enc:<metadata>:<data>:end]" in str(
```

Comment:
If we had custom exceptions, these tests wouldn't need to be so reliant on exact error message wording